src/__tests__/customFileParserTest/integrationTest.ts

```diff
@@ -73,39 +72,31 @@ describe('customFileIntegrationTest', () => {
     if (sectorMetadata.arrayCount !== 0) {
       fileReader.readUncompressedValues();
     }
-    const geometryIndexHandlers = fileReader.readSectorGeometryIndexHandlers(fileBuffer.byteLength);
+    const compressedPrimitiveData = fileReader.readCompressedGeometryData(fileBuffer.byteLength).primitives;
 
     // Check that the sector has geometries. If it doesn't, run this test on a different file.
-    expect(geometryIndexHandlers.length).toBeGreaterThan(0);
-
-    geometryIndexHandlers.forEach(geometryIndexHandler => {
-      expect(fileGeometries.indexOf(geometryIndexHandler.name)).not.toBe(-1);
-      expect(geometryIndexHandler.indexes).toBeDefined();
-      expect(geometryIndexHandler.geometryCount).toBeGreaterThan(0);
-      expect(geometryIndexHandler.byteCount).toBeGreaterThan(0);
-      expect(geometryIndexHandler.attributeCount).toBeDefined();
+    expect(compressedPrimitiveData!.length).toBeGreaterThan(0);
+
+    compressedPrimitiveData.forEach(compressedPrimitive => {
+      expect((filePrimitiveNames).indexOf(compressedPrimitive.type)).not.toBe(-1);
+      expect(compressedPrimitive.nodeIds).toBeDefined();
+      expect(compressedPrimitive.indexes).toBeDefined();
```

Comment:
Let's stick with `indices`, not `indexes` (same reason as matrices). This probably slipped my previous review!

Reply:
Fixed (although, damn that particular part of the english language, indices just looks wrong to me).